### DISCUSSION TITLE
Angular website with bootstrap looks ok on localhost, but carousel doesnt work properly on server

### DISCUSSION THREAD

**Discussion by joaquinm683** [twbs/bootstrap#37222] [General]
I have a school project with angular and bootstrap, everything works great in localhost, but when I deploy the website on a server like 000webhost or netlify, the carousel slides and button works  but it overlaps the previous  slide  till it disappear, instead of sliding the "previous"  to the left while bringing the new one from the right.

Any idea what is happening?

Thanks

**Comment by glopezfrutos** [comment]
Do you have any errors when you build?

**Reply by joaquinm683** [reply]
The only Issue I see in the web is     ":DevTools failed to load source map: Could not load content for chrome-extension://gighmmpiobklfepjocnamgkkbiglidom/browser-polyfill.js.map: System error: net::ERR_FILE_NOT_FOUND"

In happens with all my projects with the carousel, they are ok in local, but when deployed to a hosting carousel animation wont work :/

Thanks

**Reply by glopezfrutos** [reply]
I have the same problem. I opened another discussion, please check out if you find the same error messages than me when you run the build: https://github.com/twbs/bootstrap/discussions/37347

**Reply by lovish53** [reply]
@joaquinm683  Hey recently i am also working on a project and found same carousel overlap issue. Have you got any solution to this problem. If yes, can you please share the solution with me?

**Reply by glopezfrutos** [reply]
Don't remember, I guess I used CoreUI: https://coreui.io/angular/docs/components/carousel/
